### ISSUE TITLE
Feature Request: Multi-point tool with more control over point order

### ISSUE THREAD

**Issue by amstart** [fiji/fiji#190]
Hello

I use the multi-point tool to define tracks. It would be nice to be able to insert a point between the numbers 4 and 5 e.g. (such that all points n after 4 have a new value n+1 and the new point has the value 4). For instance, there could be a dialogue box asking which number should be inserted upon clicking with a modifier key.

**Comment by imagejan** [comment]
The multi-point tool is an [`imagej1`](https://github.com/imagej/imagej1) feature.
/cc @rasband

**Comment by rasband** [comment]
Have you tried using segmented lines to define the tracks? A point on a segmented line is duplicated when you shift click on it.

**Comment by amstart** [comment]
That is true,measuring with the segmented lines tool doesnt return the coordinates of the points which are part of the curve.
Coming to think about it, that would be a good alternative (is there an easy way to return the coordinates of points making up an ROI?)

**Comment by imagejan** [comment]
Sure, you can use this Groovy script for example:

```groovy
#@ ImagePlus imp

roi = imp.getRoi()
poly = roi.getPolygon()
println poly.xpoints
println poly.ypoints
```
